include/jemalloc/internal/mutex_pool_structs.h

```diff
@@ -0,0 +1,14 @@
+#ifndef JEMALLOC_INTERNAL_MUTEX_POOL_STRUCTS_H
+#define JEMALLOC_INTERNAL_MUTEX_POOL_STRUCTS_H
+
+/* This file really combines "structs" and "types", but only transitionally. */
+
+/* We do mod reductions by this value, so it should be kept a power of 2. */
+#define MUTEX_POOL_SIZE 256
```

Comment:
I was more thinking about a time in the future when 256 mutexes isn't enough.  I can easily imagine us having to manually bump this constant up over time.